test/test_mpas.py

```diff
@@ -0,0 +1,107 @@
+from uxarray._mpas import _replace_padding, _replace_zeros, _to_zero_index
+from uxarray._mpas import _read_mpas, _primal_to_ugrid, _dual_to_ugrid
+import uxarray as ux
+import xarray as xr
+from unittest import TestCase
+import numpy as np
+import os
+from pathlib import Path
+
+current_path = Path(os.path.dirname(os.path.realpath(__file__)))
+
+
+class TestMPAS(TestCase):
+    """Test suite for Read MPAS functionality."""
+
+    # sample mpas dataset
+    mpas_grid_path = current_path / 'meshfiles' / "mpas" / "QU" / 'mesh.QU.1920km.151026.nc'
+    mpas_xr_ds = xr.open_dataset(mpas_grid_path)
+
+    # fill value (remove once there is a unified approach in uxarray)
+    int_dtype = np.uint32
+    fv = np.iinfo(int_dtype).max
+
+    def test_primal_to_ugrid_execution(self):
+        """Tests execution of _primal_to_ugrid()"""
+        ugrid_ds = xr.Dataset()
+        _primal_to_ugrid(self.mpas_xr_ds, ugrid_ds)
```

Comment:
Aren't these tests up here duplicating some work done by tests below? Do we need separate tests for execution and correctness since correctness will also fail if its execution fails?